tutorials/09-variational-inference/09_variational-inference.jmd

```diff
@@ -564,7 +596,10 @@ Interestingly the squared difference between true- and mean-prediction on the te
 ```julia
 z = rand(q, 1000);
 preds = hcat(
-    [unstandardize(prediction(z[:, i], sym2range, test), data.MPG) for i in 1:size(z, 2)]...
+    [
+        unstandardize(prediction(z[:, i], sym2range, test), train_unstandardized.MPG) for
+        i in 1:size(z, 2)
+    ]...,
 );
```

Comment:
Better:
```suggestion
preds = mapreduce(hcat, eachcol(z)) do zi
    return unstandardize(prediction(zi, sym2range, test), train.MPG)
end
```

Reply:
Cool, the `mapreduce...do...` is much simpler!

Reply:
It also also much more efficient, in particular for large arrays: Generally one should not try to splat large arrays.

Reply:
Great to know. So the savings are mainly due to `mapreduce`?

Reply:
Due to avoiding splatting and since mapreduce has optimized implementations for hcat.
BTW it seems the suggestion was not applied?

Reply:
hmm...strange, I did apply all of the suggestions. Now should be applied.

Reply:
Still missing on Github it seems.

Reply:
Yes, making the commit soon.